pkg/trace/obfuscate/obfuscate.go

```diff
@@ -44,6 +45,25 @@ type Config struct {
 
 	// Redis enables obfuscatiion of the "memcached.command" tag for spans of type "memcached".
 	Memcached bool
+
+	// Set to 1 if enabled 0 if not. We're using an integer
```

Comment:
I feel it might be neater to use the standard way of documenting, which is to start with the identifier name, even if this is unexported. Also I'd like to suggest using the name `literalEscapes` because it is a better indication of what we are doing. How about something like:
```go
// literalEscapes reports whether we should treat escape characters literally or as escape characters.
// A non-zero value means 'yes'. Different SQL engines behave in different ways and the tokenizer needs
// to be generic.
// Not safe for concurrent use.
```
I noticed that the word "reports" is many times used in the Go standard library for boolean values. Arguably this isn't a boolean value.